components/formats-bsd/src/loci/formats/in/FakeReader.java

```diff
@@ -1083,9 +1083,9 @@ private int populateSPW(MetadataStore store, int screens, int plates, int rows,
     final XMLMockObjects xml = new XMLMockObjects();
     OME ome = null;
     if (screens==0) {
-      ome = xml.createPopulatedPlate(plates, rows, cols, fields, acqs);
+      ome = xml.createPopulatedPlate(plates, rows, cols, fields, acqs, false);
```

Comment:
Should the reader provide the option to call this method with w`ithMicrobeam=true`?  Likewise for createPopulatedScreen?